### PR TITLE
Improve and style playground

### PR DESCRIPTION
**Pull request by gasiortomasz**

### CODE REVIEW

File: universal-login-react/src/ui/Notifications/EmojiForm.tsx [hunk 1]
```diff
@@ -41,8 +42,8 @@ export const EmojiForm = ({sdk, publicKey, securityCodeWithFakes}: EmojiFormProp
       <EmojiPlaceholders code={enteredCode} onEmojiClicked={onEmojiRemove} />
       <EmojiPanelWithFakes securityCodeWithFakes={securityCodeWithFakes} onEmojiClicked={onEmojiAdd} />
       <div className="notification-buttons-row">
-        <button onClick={() => console.log('rejecrted lcicked')} className="notification-reject-btn">Reject</button>
-        <button onClick={() => confirmWithCodeCheck(publicKey)} className="btn btn-secondary btn-confirm">Confirm</button>
+        <button onClick={() => console.log('Rejected clicked')}>Reject</button>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiForm.tsx]: Perhaps showing that in UI is a better idea than on developer console.

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx [hunk 2]
```diff
@@ -8,15 +8,15 @@ interface EmojiPanelWithFakesProps {
 
 export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked}: EmojiPanelWithFakesProps) => {
   const emojis = securityCodeWithFakes.map((code: number, index: number) => (
-    <li key={`securityCodeWithFakes_${index}`} onClick={() => onEmojiClicked(code)} >
+    <li style={{margin: '10px'}} key={`securityCodeWithFakes_${index}`} onClick={() => onEmojiClicked(code)} >
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: inline style? bleh!

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx [hunk 3]
```diff
@@ -8,15 +8,15 @@ interface EmojiPanelWithFakesProps {
...
       <Emoji code={code}/>
     </li>
   ));
 
   return (
     <div>
       <p>Security code</p>
-      <ul style={{display: 'flex', listStyle: 'none'}}>
+      <ul style={{display: 'flex', listStyle: 'none', flexWrap: 'wrap'}}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: inline style? bleh!

File: universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ interface EmojiPlaceholdersProps {
 
 export const EmojiPlaceholders = ({code, onEmojiClicked}: EmojiPlaceholdersProps) => {
   const placeholders = code.map((code: number, index: number) => (
-    <li key={`emoji_placeholder_${index}`}  onClick={() => onEmojiClicked(index)} >
+    <li style={{margin: '10px'}} key={`emoji_placeholder_${index}`}  onClick={() => onEmojiClicked(index)} >
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx]: #inlinestyle

File: universal-login-react/src/ui/WalletSelector/EmojiPanel.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ interface EmojiPanelProps {
 
 export const EmojiPanel = ({code}: EmojiPanelProps) => {
   const emojis = code.map((code: number, index: number) => (
-    <li key={`emojiPanel_${index}`}>
+    <li style={{margin: '10px'}} key={`emojiPanel_${index}`}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/WalletSelector/EmojiPanel.tsx]: #inlinestyle

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx (hunk 2 above)
Comment (marekkirejczyk) [reply universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: btw that should be addressed in css as sth like:
```css
ul.universal-login-default  li {
  margin: 10px
}
```

btw do we have a wrapping `ul` element for those `li`'s?

File: universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ interface EmojiPlaceholdersProps {
...
       <Emoji code={code}/>
     </li>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx]: Wrapping ul missing

File: universal-login-react/src/ui/commons/NavigationColumn.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+      E2E flow
+      <li><Link className="playground-navigation-link" to="/onboarding">Onboarding</Link></li>
+    </ul>
+    <ul className="playground-navigation-list">
+      Atomic components
+      <li><Link className="playground-navigation-link" to="/walletselector">Wallet Selector</Link></li>
+      <li><Link className="playground-navigation-link" to="/connecting">Connecting</Link></li>
+      <li><Link className="playground-navigation-link" to="/topup">Topup</Link></li>
+      <li><Link className="playground-navigation-link" to="/recover">Recover</Link></li>
+      <li><Link className="playground-navigation-link" to="/settings">Settings</Link></li>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/commons/NavigationColumn.tsx]: unnecessary class, you can style with parent class `playground-navigation-list`, see below

File: universal-login-react/src/ui/styles/playground.css [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  padding: 20px;
+  background: lightskyblue;
+}
+
+.playground-navigation-list {
+  list-style: none;
+  padding: 0;
+}
+
+.playground-navigation-link {
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/styles/playground.css]:
```css
ul.playground-navigation-list li a {
 ...
}
```

File: universal-login-react/src/App.tsx [hunk 9]
```diff
@@ -1,66 +1,73 @@
...
-            sdk={sdk}
-            domains={['mylogin.eth']}
-          />
-          <EmojiPanel code={securityCode}/>
+    <BrowserRouter>
+      <div className="playground">
+        <NavigationColumn/>
+        <div className="playground-content">
+          <Switch>
+            <Route exact path="/" render={() => (<p style={{width: '80%'}}>Welcome to Universal Login</p>)}/>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/App.tsx]: `<p className="title">...</p>`

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx [hunk 10]
```diff
@@ -16,7 +16,7 @@ export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked}: Emo
   return (
     <div>
       <p>Security code</p>
-      <ul style={{display: 'flex', listStyle: 'none'}}>
+      <ul className={'universal-login-emoji-default'}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: That is going to be a bit more complex:
- we should use pattern started with WalletSelector
- we could rename `getWalletSelectorClass(className)` -> `getClassNameFor(customClassName)`
- we can add class to `<div>` above, ie: `<div className='universal-login-emoji'/>
- now we select this components in css like this:
  - Generic styles: `.universal-login-emoji ...`
  - Default styles:  `.universal-login-emoji .universal-login-default ...`
  - Custom styles: `.universal-login-emoji .some-custom-class ...`

Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: #customizableclasses

File: universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx [hunk 11]
```diff
@@ -14,7 +14,7 @@ export const EmojiPlaceholders = ({code, onEmojiClicked}: EmojiPlaceholdersProps
   ));
 
   return (
-    <ul style={{display: 'flex', listStyle: 'none'}}>
+    <ul className={'universal-login-emoji-default'}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx]: #customizableclasses

**Comment by marekkirejczyk** [comment]
I'll merge this. Let's work on CSS classes in the next PR.

**Pull request merged**